src/Microsoft.AspNet.Proxy/ProxyMiddleware.cs

```diff
@@ -50,7 +50,7 @@ public ProxyMiddleware([NotNull] RequestDelegate next, [NotNull] ProxyOptions op
 #if DNX451
             _httpClient = new HttpClient(_options.BackChannelMessageHandler?? new HttpClientHandler());
 #else
-            _httpClient = new HttpClient(_options.BackChannelMessageHandler?? new Net.Http.Client.ManagedHandler());
+            _httpClient = new HttpClient(_options.BackChannelMessageHandler?? new HttpClientHandler());
```

Comment:
Note this code is now the same as line 51, so you don't need the #if DNX451 anymore.